assets/dev/js/editor/controls/behaviors/scrubbing.js

```diff
@@ -0,0 +1,150 @@
+const SCRUB_REGULAR = 'UPDATE-VALUE';
+const SCRUB_ENHANCED = 'UPDATE-VALUE-ENHANCED';
+const SKIP_SCRUB = 'SKIP-UPDATE-VALUE';
+
+export default class Scrubbing extends Marionette.Behavior {
+	checkIntentTimeout = null;
+
+	skipperCount = 0;
+
+	constructor( ...args ) {
+		super( ...args );
+
+		const userOptions = this.getOption( 'scrubSettings' ) || {};
+
+		this.scrubSettings = {
+			intentTime: 600,
+			skipperSteps: 10,
+			enhancedNumber: 10,
+			scrubbingClass: 'e-scrubbing',
+			scrubbingElementClass: 'e-scrubbing-element',
+			scrubbingOverClass: 'e-scrubbing-over',
+			...userOptions,
+		};
+	}
+
+	ui() {
+		return {
+			input: 'input',
+			label: 'label',
+		};
+	}
+
+	events() {
+		return {
+			'mousedown @ui.input': 'onMouseDownInput',
+			'mousedown @ui.label': 'onMouseDownLabel',
+			'mouseenter  @ui.label': 'onMouseEnterLabel',
+		};
+	}
+
+	scrub( input, movementEvent ) {
+		// Determine the updating rhythm.
+		const movementType = this.getMovementType( movementEvent );
+
+		if ( SKIP_SCRUB === movementType ) {
+			return;
+		}
+
+		// Actually change the input by scrubbing.
```

Comment:
redundant comment

Reply:
I can see that you don't like comments. I disagree with you, but OK!

Reply:
(& DONE )